src/Microsoft.Extensions.Logging.Testing/XunitLoggerFactoryExtensions.cs

```diff
@@ -22,6 +22,12 @@ public static ILoggingBuilder AddXunit(this ILoggingBuilder builder, ITestOutput
             return builder;
         }
 
+        public static ILoggingBuilder AddXunit(this ILoggingBuilder builder, ITestOutputHelper output, LogLevel minLevel, DateTimeOffset? utcNow)
```

Comment:
That's the advantage of using DateTimeOffset that you need not specify the contract with parameter name like `utcNow` here. Let the user give whatever value(either with timezone or utc) they want, you could always convert it to universal time when actually using it.

Reply:
I guess this is poor naming since it doesn't actually need to be utcNow. I should just call it log start.